daprdocs/content/en/operations/security/mtls.md

```diff
@@ -199,6 +199,17 @@ kubectl rollout restart statefulsets/dapr-placement-server -n <DAPR_NAMESPACE>
 
 4. Restart your Dapr applications to pick up the latest trust bundle.
 
+{{% alert title="Recommended CLI command to rotate certificates with new ones" color="success" %}}
+The below CLI command can be used to renew root and issuer certificates of your Kubernetes cluster. It also gives an option to provide validity for the new certificates in number of days. 
+
+The 2nd variant of this command provides `--private-key` flag to support use of `private root key` of your existing deployed certificate.
```

Comment:
It is hard to separate out the kubectl approach and the Dapr CLI approach here. Can you add headings for each so that these are separated? For example "Custom root certificate upgrade using kubectl" and "Custom root certificate upgrade using CLI"